### ISSUE TITLE
Container not working with Orbstack, for some reason!

### ISSUE THREAD

**Issue by Abcmsaj** [itzg/docker-minecraft-bedrock-server#401]
Swapped from Docker Desktop to Orbstack on my Mac recently because I was constantly getting issues with Docker Desktop where all of my containers would shut down and then restarting the Docker Engine would require me to do a factory reset... not had that issue with Orbstack yet! However, I have noticed that when I use Orbstack, the logs show that my minecraft server downloads, and even says "server started" and port 19132 is open - but I cannot connect to the world.

Using the same container in Docker Desktop, there are no issues... Not really sure what the problem is as Orbstack is meant to be a drop-in replacement for Docker Desktop, and all of my other containers migrated from Docker Desktop with no issues. Any help appreciated, thanks!

**Comment by itzg** [comment]
I'm definitely hoping someone has advice for you. 

Do you mind if I convert this to a discussion since this is not an issue/bug that can fixed by the image?

**Comment by Abcmsaj** [comment]
Yeah apologies! I raised a bug in the Orbstack Github for anybody else with a similar issue. For now, I'm just running bds in Docker Desktop and every other container in Orbstack and it seems to be okay, just a risk of it using a bit too much resource longer term: https://github.com/orbstack/orbstack/issues/1058
